lib/vdc/SliceGridAlongPlane.cpp

```diff
@@ -163,7 +168,15 @@ void populateData(
             double y = yStart + i*delta.y;
             glm::tvec3<double, glm::highp> samplePoint = origin + x*axis1 + y*axis2;
             VAPoR::CoordType p = {samplePoint.x, samplePoint.y, samplePoint.z};
-            dataValues.get()[index] = grid->GetValue(p);
+
+            if ( p[0] <= min[0] || p[0] >= max[0] ||
```

Comment:
Are you finding it necessary to explicitly compare p with the extents of the Grid object? Grid::GetValue(p) should return missingValue if p is outside of the domain.If it is not then there is a bug that should be reported.

Reply:
This is checking whether the sample point is outside of the Box, no the domain.

Reply:
Understood. Since both @StasJ and I were confused about this and explicit comment might be helpful.

A small nit, perhaps, but do you really want to (need to) exclude points on the boundary (e.g. using <= instead of <)? I doubt that we are completely consistent about this, and in most instances it may not matter, but I think elsewhere points that are on the boundary are considered "valid".

Reply:
Yes, I agree and I'm going to write something up to put in the file.

I can go ahead and remove the '='.  These were added during debugging.  They had no effect but weren't removed.